test_ffi/tests/test.js

```diff
@@ -527,6 +546,25 @@ console.log(
 const view = new Deno.UnsafePointerView(dylib.symbols.static_ptr);
 console.log("Static ptr value:", view.getUint32());
 
+let rect = dylib.symbols.make_rect(10, 20, 100, 200);
+dylib.symbols.print_rect(rect);
+dylib.symbols.print_rect(new Float64Array([20, 20, 100, 200]));
```

Comment:
Might also need tests for nonblocking `print_rect`.

Reply:
Good catch... seems like it's failing.